spec/fixtures/goobi/metadata/2012315/meta.xml

```diff
@@ -53,34 +53,34 @@
    <mets:fileSec>
       <mets:fileGrp USE="LOCAL">
          <mets:file ID="FILE_0001" MIMETYPE="">
-            <mets:FLocat LOCTYPE="URL" xlink:href="file:///Users/mark/_no_backup_/g2g/goobi/metadata/2012315/images/TEST0003_media/1053442.jpg"/>
+            <mets:FLocat LOCTYPE="URL" xlink:href="file:///spec/fixtures/goobi/metadata/2012315/images/TEST0003_media/1053442.jpg"/>
```

Comment:
technically (and we're being pretty technical at the moment) file:// is the protocol and /Users/mark was the absolute path.  What you are saying here is that /spec is an absolute path, which it is not and instead these should be `file://spec/fixtures/gooobi/metadata...`. and that means the join you had before was doing the right thing ¯\_(ツ)_/¯ however, since we agreed we are likely changing this again, I think that's probably not a big deal.

Reply:
AAaaaaaah, I was wondering why there was a triple slash after file, that makes sense.